src/ubx.cpp

```diff
@@ -140,36 +140,21 @@ GPSDriverUBX::configure(unsigned &baudrate, const GPSConfig &config)
 			cfgValset<uint8_t>(UBX_CFG_KEY_CFG_UART1_DATABITS, 0, cfg_valset_msg_size);
 			cfgValset<uint8_t>(UBX_CFG_KEY_CFG_UART1_PARITY, 0, cfg_valset_msg_size);
 			cfgValset<uint8_t>(UBX_CFG_KEY_CFG_UART1INPROT_UBX, 1, cfg_valset_msg_size);
-			cfgValset<uint8_t>(UBX_CFG_KEY_CFG_UART1INPROT_RTCM3X, _output_mode == OutputMode::RTCM ? 0 : 1,
-					   cfg_valset_msg_size);
 			cfgValset<uint8_t>(UBX_CFG_KEY_CFG_UART1INPROT_NMEA, 0, cfg_valset_msg_size);
 			cfgValset<uint8_t>(UBX_CFG_KEY_CFG_UART1OUTPROT_UBX, 1, cfg_valset_msg_size);
-
-			if (_output_mode != OutputMode::GPS) {
-				cfgValset<uint8_t>(UBX_CFG_KEY_CFG_UART1OUTPROT_RTCM3X, 1, cfg_valset_msg_size);
-			}
-
 			cfgValset<uint8_t>(UBX_CFG_KEY_CFG_UART1OUTPROT_NMEA, 0, cfg_valset_msg_size);
 			// TODO: are we ever connected to UART2?
 
-			// USB
```

Comment:
We need to check if connecting via USB still works

Reply:
I guess I don't understand. Those settings were not removed, just moved until after we know what kind of a ublox device were are connected to. If it is M10 then those values are not available.

Reply:
These settings were there because they might be needed for initialization (e.g. if ubx input is disabled). I know it's a chicken-egg problem at this point. So we need to test the changes.
Can you do so? You can use SITL and start the driver on your USB tty with a USB GPS attached.

Reply:
Unfortunately I don't have a way to test that. Another option is to just send those USB configuration keys before anything else and not require an ACK. M10 will just NACK that but it won't cause any harm.

Reply:
I tested it quickly with an F9P, it still works. Can you add a comment here that we configure USB below?

Reply:
Done!